obs/installation-images.spec

```diff
@@ -74,7 +74,7 @@ ExclusiveArch:  do_not_build
 %define net_repo https://download.opensuse.org/distribution/leap/%{the_version}/repo/oss
 %else
 %define with_exfat 1
-%ifarch %arm aarch64 ppc64 ppc64le riscv64
+%ifarch %arm aarch64 ppc64 ppc64le riscv64 s390x
```

Comment:
I cannot see s390x listed in https://download.opensuse.org/ports/ and the `the_arch`variable is not set accordingly so I guess it should be also be set to `zsystems` isn't it?

Reply:
but it is rpm arch, and that is I think correct. E.g. See https://build.opensuse.org/package/view_file/YaST:SLE-15:SP3/yast2-reipl/yast2-reipl.spec?expand=1 ExclusiveArch part

Reply:
ah, now I get it and agree with you. It needs to set the_arch correctly, otherwise expansion is wrong

Reply:
@skriesch So please adjust also setting the_arch around line 60

Reply:
Thank you for this qualified review. zsystems is added as the_arch for s390x now. Have a happy new year!

Reply:
Thanks for taking care and ...

:fireworks: :tada: Happy New Year :tada: :fireworks: